cmd/juju/cloud/addcredential_test.go

```diff
@@ -934,3 +938,32 @@ User 'juju add-cloud somecloud -c controller' to add your cloud to the controlle
 `[1:]
 	s.assertAddedCredentialForCloud(c, "somecloud", stdout, "", false)
 }
+
+func (s *addCredentialSuite) TestAddRemoteCloudPromptForController(c *gc.C) {
+	s.api.clouds = func() (map[names.CloudTag]jujucloud.Cloud, error) {
+		return map[names.CloudTag]jujucloud.Cloud{
+			names.NewCloudTag("remote"): {
+				Name:      "remote",
+				Type:      "gce",
+				AuthTypes: []jujucloud.AuthType{jujucloud.JSONFileAuthType},
+			},
+		}, nil
+	}
+	stdout := `
+Do you want to add a credential to current controller "controller"? (Y/n): 
+Enter credential name: 
+Using auth-type "jsonfile".
+
+Enter path to the .json file containing a service account key for your project
+(detailed instructions available at https://discourse.jujucharms.com/t/1508).
+Path: 
+Credential "blah" added locally for cloud "remote".
+
+`[1:]
+	stderr := `
+Using  remote cloud "remote" from the controller to verify credentials.
```

Comment:
Yeah, so as per previous comment
`Using  cloud "remote" from the controller to verify credentials.`
reads way better IMO